src/Field/WfsSearch/WfsSearch.jsx

```diff
@@ -0,0 +1,389 @@
+     * rendered for each feature.
+     */
+    renderOption: feature => {
+      return (
+        <Option key={feature.id}>
+          {feature.properties.name ? feature.properties.name : feature.id}
+        </Option>
+      );
+    },
+    /**
+     * The default onSelect method if no onSelect prop is given. It zooms to the
+     * selected item.
+     *
+     * @param {object} feature The selected feature as returned by the server.
+     * @param {ol.map} olMap The openlayers map that was passed via prop.
+     */
+    onSelect: (feature, olMap) => {
+      if (feature) {
+        const olView = olMap.getView();
+        const geoJsonFormat = new OlFormatGeoJSON();
+        const olFeature = geoJsonFormat.readFeature(feature);
+        const geometry = olFeature.getGeometry();
+
+        if (geometry) {
+          olView.fit(geometry, {
+            duration: 500
+          });
+        }
+      }
+    },
+    style: {
+      width: 200
+    }
+  }
+
+  /**
+   * Create the WfsSearch.
+   *
+   * @param {Object} props The initial props.
+   * @constructs WfsSearch
+   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchTerm: '',
+      data: []
+    };
+    this.onUpdateInput = this.onUpdateInput.bind(this);
+    this.onMenuItemSelected = this.onMenuItemSelected.bind(this);
+  }
+
+  /**
+   * Called if the input of the AutoComplete is being updated. It sets the
+   * current inputValue as searchTerm and starts a search if the inputValue has
+   * a length of at least `this.props.minChars` (default 3).
+   *
+   * @param {String|undefined} value The inputValue. Undefined if clear btn
+   *                                      is pressed.
+   */
+  onUpdateInput(value) {
+    this.setState({
+      data: []
+    });
+
+    if (value) {
+      this.setState({
+        searchTerm: value || ''
+      }, () => {
+        if (this.state.searchTerm.length >= this.props.minChars) {
+          this.doSearch();
+        }
+      });
+    }
+  }
+
+  /**
+   * Perform the search.
+   * @private
+   */
+  doSearch() {
+    const {
+      featureNS,
+      featurePrefix,
+      featureTypes,
+      geometryName,
+      maxFeatures,
+      outputFormat,
+      propertyNames,
+      srsName,
+      baseUrl,
+      additionalFetchOptions
+    } = this.props;
+
+    const options = {
+      featureNS,
+      featurePrefix,
+      featureTypes,
+      geometryName,
+      maxFeatures,
+      outputFormat,
+      propertyNames,
+      srsName,
+      filter: this.createFilter()
+    };
+
+    const wfsFormat = new OlFormatWFS();
+    const featureRequest = wfsFormat.writeGetFeature(options);
+    this.setState({fetching: true});
+
+    fetch(`${baseUrl}`, {
+      method: 'POST',
+      credentials: additionalFetchOptions.credentials
+        ? additionalFetchOptions.credentials
+        : 'same-origin',
+      body: new XMLSerializer().serializeToString(featureRequest),
```

Comment:
I've in mind that there existed some problems using `XMLSerializer` in Firefox.

Did you check this also other browsers like FF, Edge, ..?

Reply:
No i didn't nice hint. I'll check this.

Reply:
After a quick check, it works as aspected in FireFox and InternetExpolrer.